autoload/sy.vim

```diff
@@ -106,7 +106,16 @@ endfunction
 
 " #buffer_is_active {{{1
 function! sy#buffer_is_active()
```

Comment:
Why not simplify it to this?

```vim
function! sy#buffer_is_active(...)
  let bufnr = a:0 ? a:1 : bufnr('')
  return !empty(getbufvar(bufnr, 'sy'))
endfunction
```

I don't see the need for an extra function.

Reply:
sure, good trick. 'sy#buffer_is_active_bnr' was for clear and specific to 'bnr'.
it's up to you, it's ok. if that is preferred, pls modify that in `sy#util#refresh_windows()` as well.
pls let me know, i can force push a new one as that, or you modify it here directly.. or let it be?

Reply:
nevermind, force push a new one per your advice.